ndk/src/relay/connectivity.test.ts

```diff
@@ -0,0 +1,33 @@
+import { NDKRelayConnectivity } from "./connectivity.js";
+import { NDKRelay, NDKRelayStatus } from "./index.js";
+
+describe("NDKRelayConnectivity", () => {
+    let ndkRelayConnectivity: NDKRelayConnectivity;
+    let ndkRelayMock: NDKRelay;
+    let relayConnectSpy: jest.SpyInstance;
+    beforeEach(() => {
+        ndkRelayMock = new NDKRelay("ws://localhost");
+        ndkRelayConnectivity = new NDKRelayConnectivity(ndkRelayMock);
+        // Mock the connect method on nostr tools relay
+        relayConnectSpy = jest.spyOn(ndkRelayConnectivity.relay, "connect").mockResolvedValue();
+    });
+
+    afterEach(() => {
+        jest.restoreAllMocks();
+    });
+
+    it("should connect and disconnect from the relay", async () => {
```

Comment:
not a big deal, but let's always be assertive in the comments

"should connect and disconnect from the relay" => "connects and disconnects from the relay"

Reply:
Old habit from Ruby-land 🙈